torch/nn/utils/parametrize.py

```diff
@@ -247,7 +244,7 @@ def right_inverse(self, value: Tensor) -> None:
 
     def forward(self) -> Tensor:
         # Unpack the originals for the first parametrization
-        if self.ntensors == 1:
+        if self.is_tensor == 1:
```

Comment:
These are bool checks no? You don't want the == 1 ?

Reply:
Oooopsss. Sorry for that and thanks for the catch!